projects/standalone-app-e2e/src/integration/app.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { getGreeting } from '../support/app.po';
+
+describe('standalone-app', () => {
+  beforeEach(() => cy.visit('/'));
+
+  it('should display welcome message', () => {
+    // Custom command example, see `../support/commands.ts` file
+    cy.login('my-email@something.com', 'myPassword');
```

Comment:
Great idea to add a new app to test these new features out.

If it's ok for you, I can update the e2e test in a follow-up PR.
Because now, this `login` command confused me because I didn't see a login prompt in the app. 
I would like to clean the cypress setup up, and also make sure that the feature is loaded correctly after navigating to the page.

Reply:
Yep, that's fine with me